frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -260,6 +260,15 @@ class Worksheet extends React.Component {
             worksheet_uuid,
         )
             .done(() => {
+                Object.keys(this.state.checkedBundles).forEach((uuid) => {
```

Comment:
I changed `.map` to `.forEach` because otherwise it caused an eslint error (since the function doesn't return anything)